src/components/Splashscreen/index.js

```diff
@@ -1,11 +1,46 @@
-import { connect } from 'react-redux';
-import { actionCreators as navActionCreators } from '../../ducks/navigation';
-import Splashscreen from './presenter';
-
-function mapDispatchToProps(dispatch) {
-  return {
-    onEnd: () => dispatch(navActionCreators.doReplace('splashscreen', { key: 'index' })),
-  };
-}
+// @flow
+import React from 'react';
+import {
+  Text,
+  View,
+} from 'react-native';
+import AnimatedLogo from '../common/AnimatedLogo';
+import styles from './style';
+
+type Props = {
+  navigation: Object,
+};
+
+type State = {
+  title: String,
+};
+
+export default class Splashscreen extends React.Component<Props, State> {
+  static navigationOptions = {
+    title: 'Splash',
+  }
 
-export default connect(null, mapDispatchToProps)(Splashscreen);
+  constructor(props) {
+    super(props);
+
+    this.goToNext = this.goToNext.bind(this);
+  }
+
+  goToNext() {
+    if (this.props.navigation != null) {
```

Comment:
You can safely ignore this check, Navigator will always pass `navigation` to the component